### DISCUSSION TITLE
I can´t start a server (error unknown certificate type) on openssl 3.0.x

### DISCUSSION THREAD

**Discussion by javidp7** [open-quantum-safe/oqs-provider#455] [Q&A] [answered]
I am following the guide to start the test server but i get this error all the time (i have use dilithium3): 
![image](https://github.com/user-attachments/assets/b67a3558-b8c2-4b59-a9ad-954b92e65184)

I have execute the runtest.sh and everything is ok, and the oqsprovider is active:
![image](https://github.com/user-attachments/assets/52ae0381-b1f5-4513-bf5c-a1d06a38c814)
![image](https://github.com/user-attachments/assets/b3f16110-a819-4b41-a2b2-a6436cb9cee2)
![image](https://github.com/user-attachments/assets/9bee992c-83a2-4848-859c-03c84ab05fc6)

And this is the change i made to openssl.cnf (I have changed the CipherString to 0 because i was getting the error my certificate is too weak)
![image](https://github.com/user-attachments/assets/b2bf4736-8466-460b-9827-6fc7292dea7f)

**Comment by baentsch** [comment, marked as answer]
This cannot work, see https://github.com/open-quantum-safe/oqs-provider/?tab=readme-ov-file#note-on-openssl-versions: Your openssl version is too old to support provider based signature algorithms (like dilithium). Upgrade `openssl` to 3.2 or higher and it will work.
